Formula/mypy.rb

```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/JukkaL/mypy.git",
-    :tag => "v0.4.2",
-    :revision => "b8a2df91707bc3d5c9c3677c8017b614e41d017a"
+      :tag => "v0.4.3",
```

Comment:
For invocations of `./configure` we clearly prefer to align the options. For other tools and binaries where there's maybe less structure and an intermediate subcommand, this rule doesn't work as well. (I haven't seen a style that is both consistent and looks beautiful in all circumstances.)

The `./configure` style is also responsible for why [none of our method invocations are checked for proper argument alignment](https://github.com/Homebrew/brew/blob/b6772c1fde5878fdb492aad8b4f9c0bc10284a29/Library/.rubocop.yml#L102-L104) and why some of those `url` examples got away with no indentation at all. The only modes RuboCop supports is either:

- align with the first argument in the first line or
- indent one step (two spaces)

for any arguments that don't fit on the first line. I don't have a strong preference, should we consider enabling this check, as long as we indent method arguments at all.